examples/express-ts/src/middlewares.ts

```diff
@@ -1,28 +1,48 @@
 import * as store from '@leapfrogtechnology/async-store';
 import { Request, Response, NextFunction } from 'express';
 
+import * as logger from './logger';
+import { doSomething } from './service';
+
 /**
- * Middleware to set the request context `x-id` on the async store.
+ * Middleware to set query params `a` and `b` on async-store.
  *
  * @returns {(req, res, next) => void}
  */
-export function requestContext() {
+export function requestParams() {
   return (req: Request, res: Response, next: NextFunction) => {
-    store.set({ 'x-id': req.header('x-id') });
+    const a = req.query.a;
+    const b = req.query.b;
+
+    store.set({
+      a: parseFloat(a) || 0,
```

Comment:
No need to `parseFloat`. This is just an example and we are responsible of sending the query params. Thus, I don't think we need it.

```js
store.set({ a, b })
```